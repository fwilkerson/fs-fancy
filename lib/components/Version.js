import React from 'react';

const Version = props => {
  return <span>version: {props.version}</span>;
};

export default Version;
