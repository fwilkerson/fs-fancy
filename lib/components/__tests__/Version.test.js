import React from 'react';
import { create } from 'react-test-renderer';

import Version from '../Version';

describe('Version SFC', () => {
  const testProps = {
    version: '0.0.0'
  };

  it('renders without crashing', () => {
    const tree = create(<Version {...testProps} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
