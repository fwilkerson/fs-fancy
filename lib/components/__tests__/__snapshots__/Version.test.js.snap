// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Version SFC renders without crashing 1`] = `"version: 0.0.0"`;
